app/src/userpages/components/StreamPage/Show/ConfigureView/index.jsx

```diff
@@ -241,25 +244,31 @@ export class ConfigureView extends Component<Props, State> {
                 }
                 <div className={styles.settings}>
                     {/* eslint-disable jsx-a11y/label-has-associated-control */}
+                    {stream && stream.autoConfigure !== undefined &&
+                        <Fragment>
```

Comment:
Fragment should not be needed here?

Reply:
ah yep, looks like some overcooked copy pasta. Cheers